libs/blocks/table/table.js

```diff
@@ -97,6 +97,36 @@ function handleHeading(table, headingCols) {
   });
 }
 
+function handleAddOnContent(table) {
+  const addOnKey = 'ADDON';
+  const addOns = [...table.querySelectorAll('.section-row-title')]
+    .filter((row) => row.innerText.includes(addOnKey));
+  if (addOns.length) {
```

Comment:
This is getting quite "indenty", could we switch to an early return?
```suggestion
  if (!addOns.length) return;
```
Same goes for L111 & L115.

Reply:
Okay I added early returns for all.